site/public/js/gradeable.js

```diff
@@ -195,10 +195,16 @@ function renderPeerGradeable(grader_id, gradeable, graded_gradeable, grading_dis
  * @param {boolean} componentVersionConflict
  * @returns {Promise<string>} the html for the graded component
  */
-function renderGradingComponent(grader_id, component, graded_component, grading_disabled, canVerifyGraders, precision, editable, showMarkList, componentVersionConflict) {
+function renderGradingComponent(grader_id, component, graded_component, grading_disabled, canVerifyGraders, precision, editable, showMarkList, componentVersionConflict, is_student) {
```

Comment:
There are too many arguments for this single function, consider refactor this function?

Reply:
Granted, this is hard to refactor. Maybe an attempt could be made?

Reply:
I can fit precision, editable, showMarkList, and is_student into an array instead, but this might cause slight confusions for future developers, what do you think?